client/packages/cli/index.js

```diff
@@ -637,25 +666,35 @@ async function saveConfigAuthToken(authToken) {
   return writeFile(authPaths.authConfigFilePath, authToken, "utf-8");
 }
 
-function execMagicLocalhostCallback(port) {
-  return new Promise((resolve) => {
-    const server = createServer(async (req, res) => {
-      res.setHeader("Access-Control-Allow-Origin", instantDashOrigin);
-      res.setHeader("Access-Control-Allow-Methods", "POST, OPTIONS");
-
-      const data = await readReqBody(req).catch(() => {});
-
-      if (!data?.token) {
-        res.statusCode = 400;
-        res.end();
-      } else {
-        resolve(data);
-        res.statusCode = 200;
-        res.end();
-        server.close();
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function waitForAuthToken({ secret }) {
+  for (let i = 1; i <= 24; i++) {
+    await sleep(i * 5000);
```

Comment:
Do I understand correctly, that the flow is: 

```
Wait 5 seconds, then check
If nothing, wait 10 seconds and check
If nothing, wait 15 seconds and check
```

I worry 5 seconds may be a bit too long even in the happy case, and that afterwards the experience will feel slower.

Reply:
Dropped to 1